### PR TITLE
0.2/165 select screen

### PR DESCRIPTION
**Pull request by gingerbreadpup**

### CODE REVIEW

File: src/pages/Payment.vue [hunk 1]
```diff
@@ -5,9 +5,10 @@
             <div class="content fullwidth">
                 <h1>My Voucher List</h1>
 
-                <p>You can request payment for <strong>{{ recVouchers[0].length }}</strong> vouchers.</p>
+                <p v-if="recVouchers.length > 0">You can request payment for <strong>{{ recVouchers[0].length }}</strong> vouchers.</p>
+                <p v-else>You haven't added any vouchers yet.</p>
 
-                <div class="voucher-list" id="registeredVouchers">
+                <div class="voucher-list" id="registeredVouchers" v-if="recVouchers.length > 0">
```
Comment (charlesstrange2) [inline src/pages/Payment.vue]: does that need the [0]?

**Review by charlesstrange2 — CHANGES_REQUESTED** [review]
hav a tllok at that recVouchers.length , it may need a [0]

**Comment by gingerbreadpup** [comment]
@charlesstrange2 there you go!

**Review by charlesstrange2 — APPROVED** [review]
awesome
:shipit:

**Pull request merged**